tests/suites/events.js

```diff
@@ -301,7 +301,7 @@ test('setDate: triggers change and changeDate events', function(){
 
     this.dp.setDate(new Date(2011, 2, 5));
 
-    equal(triggered_change, 2);
+    equal(triggered_change, 0);
```

Comment:
Should the change event not be called once? Or do we just want to leave out the change event and only have a changeDate event, but that would mean a BC break